bin/nrnivmodl_makefile_cmake.in

```diff
@@ -19,7 +19,7 @@ UserLDFLAGS =
 # install dirs
 bindir := ${ROOT}/@CMAKE_INSTALL_BINDIR@
 libdir := ${ROOT}/@CMAKE_INSTALL_LIBDIR@
-incdir := ${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@
+incdir := -I${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@ -I${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@/crout -I${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@/neuron -I${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@/newton -I${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@/nrncvode -I${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@/nrniv -I${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@/nrnmpi -I${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@/nrnoc -I${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@/oc -I${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@/scopmath -I${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@/sparse13
```

Comment:
I'm confused by this change. If we are now including a lot more of these directory names in the `#include <...>` statements, why do we need these here?

Reply:
because mod files still include "raw" files from the old grouped `<BUILD>/include` directory

Reply:
I suppose that is widespread in `VERBATIM` blocks, not just in the code generated by `nocmodl`?

I'm not sure it's a step forwards to introduce two different `#include` schemes, one for the library source files and one for the generated `.cpp`...

Reply:
I don't buy that; the cost/reward for updating all of ModelDB for that is such that it will never happen, in my view.

So: this seems like it's creating and locking in a new source of inconsistency between different bits of C++. Is that really worth it?

Reply:
Thoughts @pramodk @alexsavulescu @nrnhines ?

Reply:
>> I don't buy that; the cost/reward for updating all of ModelDB for that is such that it will never happen, in my view.
>
> I agree on the first part that it will never happens.

We are on the same page!

>> So: this seems like it's creating and locking in a new source of inconsistency between different bits of C++. Is that really worth it?
> 
> For the second part, I, personnaly, think, it worse it, to clean more the code and know who include what.

I understand the "inconsistency" aspects that Olli mentioned (i.e. having a single way would be better). 

I don't have _recent enough hands-on experience_ to assess the impact or propose a better solution here. Maybe @olupton you can propose what should be done here?